src/components/workAllocation/WorkAllocation.tsx

```diff
@@ -189,11 +198,11 @@ export default function WorkAllocation() {
       .label("Cost Code"),
     isRnD: Yup.boolean().required(),
     numBlocks: Yup.number().max(MAX_NUM_BLOCKANDSLIDES),
-    numSlides: Yup.number()
-      .max(MAX_NUM_BLOCKANDSLIDES)
-      .when("numBlocks", (numBlocks: any, schema: any) => {
-        if (!numBlocks) {
-          return schema.required("Number of blocks or slides required");
+    numSlides: Yup.number().max(MAX_NUM_BLOCKANDSLIDES),
+    numOriginalSamples: Yup.number().max(MAX_NUM_BLOCKANDSLIDES)
```

Comment:
Just a doubt - Is the max value for original samples same as blocks and slides?

Reply:
Good point. Ill ask Katy.